pkg/ipamfetcher/customer_test.go

```diff
@@ -9,9 +9,10 @@ import (
 	"net/url"
 	"testing"
 
-	"github.com/asecurityteam/ipam-facade/pkg/domain"
 	gomock "github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
+
```

Comment:
Nit: gofmt used to complain about empty lines here, I guess that's changed recently.

Reply:
oh yeah, we still haven't agreed on if we want to organize imports into 2 sections (standard lib, then everything else) or 3 sections (standard lib, 3rd party, refs to this repo). This one is split into 3.